src/store/storeFactory.js

```diff
@@ -112,6 +112,7 @@ export default ({namespace, listFn, paginateFn, createFn, updateFn, deleteFn, re
 					for (let d of data) {
 						state[namespace].push(d);
 					}
+					state.missing = data['federation:missing'];  // yes, accessing a property on an array
```

Comment:
The missing is not initiated in the getDefaultState method above, I could imagine that this could become an issue and lead to the issues you saw.

Reply:
Good spot!! I added it to getDefaultState and the weird behaviour indeed went away. Let's see if this fix is more permanent now 😆

Reply:
Fixed in 84f3e767873b56696b2c6bdbc029f389632b6dd5